bayesian/utils.py

```diff
@@ -172,20 +172,19 @@ def __init__(self, package, version, response):
         self._package = package
         self._version = version
         self._cves = []
-        self._nocve_versions = []
+        # Gremlin query will always return two elements for response
```

Comment:
It seems all the changes are being pulled in from my PR. You need to rebase your branch after pulling latest changes.